templates/pricing/devices.html

```diff
@@ -0,0 +1,649 @@
+{% extends "pricing/base_pricing.html" %}
+
+{% block title %}Ubuntu IoT and Devices | plans and pricing{% endblock %}
+{% block meta_description %}Canonical enables sophisticated devices to be designed, delivered and operated at scale with Ubuntu and Ubuntu Core, the all-snap solution to IoT software delivery and security.{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1uiKNaT59tk2rvjuM24vpqVblcNO90PmE4wUlq9A9Rsk/edit{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip is-bordered">
+  <div class="u-fixed-width">
+    <h1>Ubuntu IoT and device services</h1>
+    <p>Ubuntu is the new standard for embedded Linux development and the intelligent edge. Get ten years of security coverage and dedicated app stores for your smart connected devices.</p>
+    <a href="/support/contact-us" class="p-button--positive js-invoke-modal">Get in touch</a>
```

Comment:
clicking this didn't open the modal

Reply:
Sorry, missed the JS include - I've included the "general" contact form partial, there isn't one specific to devices (whereas there is for infra, for example), should we have one created?

Reply:
general is ok for now, but we should get one for this page particularly